### ISSUE TITLE
Collection show pages lack h1

### ISSUE THREAD

**Issue by alexdryden** [mlibrary/umich-arclight#143]
We borrowed your [repository cards](https://github.com/mlibrary/umich-arclight/blob/3b724496748e1361cf27cb25b5f98c4353ac2dfa/app/views/arclight/repositories/_repository.html.erb#L45C1-L52C11) for our implementation of Arclight at Illinois, and during an accessibility review found that non-index pages that use the cards (e.g., [here](https://findingaids.lib.umich.edu/catalog?f%5Blevel_sim%5D%5B%5D=Collection&f%5Brepository_sim%5D%5B%5D=University+of+Michigan+Bentley+Historical+Library)) lack an h1 element and go directly to the h2 element of the [card title](https://github.com/mlibrary/umich-arclight/blob/3b724496748e1361cf27cb25b5f98c4353ac2dfa/app/views/arclight/repositories/_repository.html.erb#L45C1-L52C11). The fix suggested by our reviewer was to bump the heading level of the headings for the horizontal, single use cards to h1 and h2, and I figured I would pass the feedback along. 

Many thanks for publishing your implementation.
